bin/proxy.js

```diff
@@ -4,15 +4,13 @@ const fs = require('fs');
 const http = require('http');
 const path = require('path');
 
-const bunyan = require('bunyan');
 const config = require('config');
 const cookie = require('cookie');
 const httpProxy = require('http-proxy');
+const pino = require('pino');
 
-const log = bunyan.createLogger({
-  name: 'proxy',
-  app: config.get('appName'),
-  serializers: bunyan.stdSerializers,
+const log = pino({
```

Comment:
Could you pass `level` here from the config?

I think it might be better to change the [request logging](https://github.com/mozilla/addons-frontend/blob/0e9d837c326c446145959607704ed9f68899690a/bin/proxy.js#L63) to a debug message. What do you think?

Reply:
> I think it might be better to change the request logging to a debug message. What do you think?

@kumar303 I like to have those logs, I read them often. We could switch them to `debug`, but then we should probably have `loggingLevel: 'debug'` for the `development` env?

Reply:
I decided to turn the debug logs on in `development` because it keeps compatibility with the current behavior and it really makes sense to have such logs in development. Otherwise, we'll never seen/read them, never. And I'd be mad if I had to restart the server to read a debug log.

Reply:
> I like to have those logs, I read them often

OK, I was on the fence about it. Let's keep them as `info`.

> I decided to turn the debug logs on in development because it keeps compatibility with the current behavior

Sure. I find it a bit noisy so I'll probably set it to `info` locally and see how it goes.

> I'd be mad if I had to restart the server to read a debug log.

That's the age old logging problem 😁 I find that *most* bugs are reproducible so I don't mind switching into debug mode only when needed. For that one time the bug is not reproducible it will be annoying, yes.